<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageEmulationDevDialog</class>
 <widget class="QDialog" name="ManageEmulationDevDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Devices</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RobloxStudio.qrc">
    <normaloff>:/images/RobloxStudio.png</normaloff>:/images/RobloxStudio.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="deviceListWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="plusButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="RobloxStudio.qrc">
         <normaloff>:/16x16/images/Studio 2.0 icons/16x16/plus_16.png</normaloff>:/16x16/images/Studio 2.0 icons/16x16/plus_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minusButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="RobloxStudio.qrc">
         <normaloff>:/16x16/images/Studio 2.0 icons/16x16/minus_16.png</normaloff>:/16x16/images/Studio 2.0 icons/16x16/minus_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RobloxStudio.qrc"/>
 </resources>
 <connections/>
</ui>
